pylab/website/migrations/account/0001_initial.py

```diff
@@ -30,11 +30,11 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='EmailConfirmation',
             fields=[
-                ('id', models.AutoField(primary_key=True, verbose_name='ID', auto_created=True, serialize=False)),
+                ('id', models.AutoField(auto_created=True, verbose_name='ID', primary_key=True, serialize=False)),
                 ('created', models.DateTimeField(default=django.utils.timezone.now, verbose_name='created')),
-                ('sent', models.DateTimeField(verbose_name='sent', null=True)),
-                ('key', models.CharField(verbose_name='key', unique=True, max_length=64)),
-                ('email_address', models.ForeignKey(verbose_name='e-mail address', to='account.EmailAddress')),
+                ('sent', models.DateTimeField(null=True, verbose_name='sent')),
+                ('key', models.CharField(unique=True, max_length=64, verbose_name='key')),
+                ('email_address', models.ForeignKey(to='account.EmailAddress', verbose_name='e-mail address')),
             ],
             options={
                 'verbose_name_plural': 'email confirmations',
```

Comment:
As I see, nothing was changed here, only parameter ordering was changed. You should try to avoid such changes in commits.